snuba/migrations/groups.py

```diff
@@ -153,6 +156,11 @@ def __init__(
         storage_sets_keys={StorageSetKey.SPANS},
         readiness_state=ReadinessState.PARTIAL,
     ),
+    MigrationGroup.GROUP_ATTRIBUTES: _MigrationGroup(
+        loader=GroupAttributesLoader(),
+        storage_sets_keys={StorageSetKey.GROUP_ATTRIBUTES},
```

Comment:
I believe this new storage set needs to be added in this PR so we can use it

Reply:
Specifically, do you mean adding it here: https://github.com/getsentry/snuba/pull/4496/commits/c3ea983704e35f503fed48a91b09fbdfd8569642

Reply:
yup